Gherkin Features/ID016_Create_Group_Of_Friends.feature

```diff
@@ -0,0 +1,48 @@
+Feature: Create a group of friends 
+
+As an OnCampus User
+  I would like to create a group of friends 
+  So that I can keep share my schedule to a group of friends that I wish
```

Comment:
the sentence seems a bit off? maybe "So that i could share a single schedule where we can see our shared events" or something like that?

Reply:
Oh mb didn't notice the typo